bin/tldr

```diff
@@ -22,11 +22,12 @@ program
   .option('-e, --random-example', 'Show a random example')
   .option('-f, --render [file]', 'Render a specific markdown [file]')
   .option('-m, --markdown', 'Output in markdown format')
-  .option('-o, --os [type]', 'Override the operating system [linux, osx, sunos, windows]')
-  .option('--linux', 'Override the operating system with Linux')
-  .option('--osx', 'Override the operating system with OSX')
-  .option('--sunos', 'Override the operating system with SunOS')
-  .option('--windows', 'Override the operating system with Windows')
+  .option('-p, --platform [type]', 'Override the current platform [android, linux, osx, sunos, windows]')
```

Comment:
From the other issue, I don't think it's worth fully dropping `-o` and `--os` given that not using either currently, and instead would add it as an [`.alias`](https://github.com/tj/commander.js/blob/master/examples/alias.js). It won't be documented, but if people still use it, :shrug:

Could maybe look to see if could add a way to detect if `--os` was used vs `--platform` and show a deprecation warning to eventually remove it.

Reply:
Thanks for the suggestion, will try out this alias method instead. (Not really familiar with NodeJS but will figure it out, if I have any doubts will reach out)